tests/test_metadata_serialization.py

```diff
@@ -92,6 +182,25 @@ def test_root_serialization(self, test_case_data: str):
         root = Root.from_dict(copy.deepcopy(case_dict))
         self.assertDictEqual(case_dict, root.to_dict())
 
+
+    invalid_metafile_attr: DataSet = {
+        "wrong length type": '{"version": 1, "length": "a", "hashes": {"sha256" : "abc"}}',
+        "length 0": '{"version": 1, "length": 0, "hashes": {"sha256" : "abc"}}',
+        "length below 0": '{"version": 1, "length": -1, "hashes": {"sha256" : "abc"}}',
+        "empty hashes dict": '{"version": 1, "length": 1, "hashes": {}}',
+        # Hashes keys cannot be strings because the JSON specification.
+        # Then, do we want to test for that?
```

Comment:
this does not seem to belong in the test data (but I also don't understand the comment)

Reply:
Oh... Sorry, it was a comment to myself and forgot to remove it.